docs/_embed/automatic_account_creation.md

```diff
@@ -6,19 +6,20 @@ redirect_from: /partner_api/
 ---
 
 # Automatic Account Creation
-![](https://cdn.zappy.app/e45cd24ed1fbd687a7294bb956440c6e.gif)
 
-*Pictured above: A Zapier partner demo site, Space by Zapier, uses Auth0 as their identity provider and has enabled Automatic Account Creation.  When a new user interacts with a Zapier embed the user is asked to consent in sharing their profile information with Zapier and an account is created on their behalf.*
+Save time and reduce friction for your users by having a Zapier account created on their behalf when using one of our produt embeds.  Users no longer need to worry about the hassle of signing up to a new service.
+
+<video controls loop src="https://cdn.zappy.app/29e94e1421ab8c4cbae09bcba07f2fdb.mp4" height="100%" width="100%"></video>
```

Comment:
Testing this locally, it works in Jekyll

Reply:
>The width of the video's display area, in [CSS pixels](https://drafts.csswg.org/css-values/#px) (absolute values only; [no percentages](https://html.spec.whatwg.org/multipage/embedded-content.html#dimension-attributes)).

width/height in HTML attributes can only take pixel values (without the `px` suffix), so these are likely being ignored?